index.html

```diff
@@ -25,11 +26,37 @@
           <h1 class="logo-heading">Fun Bus</h1>
         </div>
         <div class="col-9">
-          <nav class="nav d-flex justify-content-end mt-3">
-            <a class="nav-link active" href="#">Home</a>
-            <a class="nav-link" href="#">About Us</a>
-            <a class="nav-link" href="#">Blog</a>
-            <a class="nav-link" href="#">Contact</a>
+          <nav class="navbar navbar-expand-lg navbar-light bg-light mt-3">
```

Comment:
Your nav functionality is awesome! I dig the Contact drop down menu, the hover and active in the nav. I'm not sure if you intended your nav items to be centered more towards the right. Generally speaking, it is on the left.